### PR TITLE
Added Area function

### PR DESCRIPTION
**Pull request by samritchie**
Added a `Geometry.area()` method

### CODE REVIEW

File: GEOSwift.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (vfn) [inline GEOSwift.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist]: Can you please remove this file from the commit/pr?

Comment (vfn) [reply GEOSwift.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist]: ...or add it to an independent commit

Comment (samritchie) [reply GEOSwift.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist]: You want me to gitignore this file? Apple recommend committing it:

> Xcode 9.3 adds a new IDEWorkspaceChecks.plist file to a workspace's shared data, to store the state of necessary workspace checks. Committing this file to source control will prevent unnecessary rerunning of those checks for each user opening the workspace. (37293167)

Comment (vfn) [reply GEOSwift.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist]: I meant for this commit/PR, as it doesn't have anything to do with it. I'm happy for you to add it as a separate commit

**Pull request merged**